src/test/java/showcase/flows/OpenPageFlow.java

```diff
@@ -0,0 +1,38 @@
+package showcase.flows;
+
+import static com.codeborne.selenide.Condition.hidden;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.ex.ElementNotFound;
+
+public class OpenPageFlow
+{
+    public static void openXceptanceJobOffersPage()
+    {
+        // a helper function to open the job page and close the GDPR dialog to avoid duplicate code
```

Comment:
move comment above the function (can also changed into JavaDoc)

Reply:
done